scripts/build_wheel.py

```diff
@@ -865,6 +865,42 @@ def get_binding_lib(subdirectory, name):
             # and validating python changes in the whl.
             clear_folder(dist_dir)
 
+        # Modify requirements.txt for wheel build based on CUDA version
+        def modify_requirements_for_cuda():
+            requirements_file = project_dir / ("requirements-windows.txt"
```

Comment:
Why do we still care about Windows?

I think windows support is already removed.